src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -0,0 +1,393 @@
+  // This state is updated when the user edits the box holder email under pending changes
+  const [boxHolderEmailState, setBoxHolderEmailState] = useState(boxHolderEmail);
+  // A state for the box's zip code
+  // This state is updated when the user edits the zip code under pending changes
+  const [zipCodeState, setZipCodeState] = useState(zipCode);
+  // A state for the box's general location
+  // This state is updated when the user edits the general location under pending changes
+  const [generalLocationState, setGeneralLocationState] = useState(generalLocation);
+  // A state for the box's message
+  // This state is updated when the user edits the message under pending changes
+  const [messageState, setMessageState] = useState(message);
+  // A state for the box's message status
+  // This state is updated when the user approves or rejects the message under pending changes
+  const [messageStatusState, setMessageStatusState] = useState(messageStatus);
+  // A state for the box's launched organically state
+  // This state is updated when the user edits the launched organically field under pending changes
+  const [launchedOrganicallyState, setLaunchedOrganicallyState] = useState(launchedOrganically);
+
+  // A function that updates box information in the backend and refetches all boxes that are under review or pending changes (message status can be updated in 'under review')
+  // This method is called when the save button is clicked under pending changes
+  const updateBoxInfo = async stat => {
+    await FYABackend.put('/boxHistory/update', {
+      boxID,
+      status: stat,
+      boxHolderName: boxHolderNameState,
+      boxHolderEmail: boxHolderEmailState,
+      zipCode: zipCodeState,
+      generalLocation: generalLocationState,
+      message: messageState,
+      messageStatus: messageStatusState,
+      launchedOrganically: launchedOrganicallyState,
+    });
+
+    const requests = [fetchBoxes('under review', false), fetchBoxes('pending changes', false)];
+    await Promise.all(requests);
+  };
+
+  // A function that approves a relocation box submission and updates the backend state accordingly and then refetches all boxes (boxes can be approved from any tab)
+  const approveRelocationBoxFromUR = async id => {
+    await FYABackend.put('/boxHistory/update', {
+      boxID,
+      status,
+      boxHolderName: boxHolderNameState,
+      boxHolderEmail: boxHolderEmailState,
+      zipCode: zipCodeState,
+      generalLocation: generalLocationState,
+      message: messageState,
+      messageStatus: messageStatusState,
+      launchedOrganically: launchedOrganicallyState,
+    });
+    await FYABackend.put('/boxHistory/approveBox', {
+      boxID: id,
+    });
+    const requests = [
+      fetchBoxes('under review', false),
+      fetchBoxes('pending changes', false),
+      fetchBoxes('evaluated', false),
+    ];
+    await Promise.all(requests);
+  };
+
+  return (
+    <ChakraProvider>
```

Comment:
nit: According to the Figma design, the coloring should only be at the top of the box element (above all the input fields). Is it possible to do this? If this is not possible, no worries, but in this case please add margin/padding at the bottom between the white box data and the color.